itl/command-plugins.conf

```diff
@@ -3140,3 +3140,34 @@ object CheckCommand "rpc" {
 
 	vars.rpc_address = "$check_address$"
 }
+
+object CheckCommand "uptime" {
+        import "ipv4-or-ipv6"
```

Comment:
The import here shouldn't be necessary since the CheckCommand don't use the `$check_address$` macro.

Reply:
fixed as well :-)